### ISSUE TITLE
Add functionality: widgets to be movable in some way without using a mouse (Accessibility)

### ISSUE THREAD

**Issue by Phanoix** [tbs-sct/gcconnex#2]
Widgets need to be movable using only a keyboard, and this functionality should be made clear somewhere on pages with movable widgets.

**Comment by Troy-Lawson** [comment]
@piet0024 can this be closed?

**Comment by Phanoix** [comment]
This is currently on production, so I would say it's safe to close.
